README.md

```diff
@@ -363,7 +361,8 @@ is omitted, then the output will be returned to stdout.  Here are the sign
 >command. If the request URL is sent by email, then it is not returned by the command.
 >If the input parameter is omitted, then the command will take its input from stdin.
 >Returns to stdout the URL to be handed to the signer if the URL is not sent by email
->otherwise return nothing.
+>otherwise return nothing. Document have to be already registered otherwise it will 
```

Comment:
Document**s**

Reply:
I think it should be Document, for now, cli works only with the single document at the time.